test/index.ts

```diff
@@ -21,3 +27,34 @@ describe('Access the server', () => {
     expect(response.body.data.hello).to.be.eq('Hello, world!');
   });
 });
+
+describe('Mutation createUser', () => {
+  it('Run mutation createUser', async () => {
+    const response = await request('localhost:4000').post('/').send(queryCreateUser);
+
+    expect(response.body.data.createUser).to.have.property('id').not.undefined;
```

Comment:
verificar se o a propriedade existe não garante muita coisa... concorda?
Costumamos validar direto o conteudo da propriedade (e ai não precisamos nem ficar vendo se a propriedade existe)
`expect(response.body.data.createUser.name).to.be.eq('Neymar')`

Reply:
Concordo, irei alterar.

Porém, é mais complicado checar o valor do ID, pois ele é gerado incrementalmente, então cada vez que o teste é rodado, ele muda, e dar clear no repositório não reseta isso. Pensei em só checar se o ID contém um número

Reply:
no caso do id basta avaliar se é um número mesmo :)

Reply:
feito